docs/docs/exosphere/state-manager-setup.md

```
@@ -0,0 +1,374 @@
+- Load balancer and ingress controller
+- SSL certificates for HTTPS
+
+#### Kubernetes Deployment
+
+1. **Create namespace**:
+   ```bash
+   kubectl create namespace exosphere
+   ```
+
+2. **Create ConfigMap for configuration**:
+   ```yaml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)